### DISCUSSION TITLE
A random reddit discussion

### DISCUSSION THREAD

**Discussion by hungpham3112** [JuliaLang/Pkg.jl#3556] [General]
Hi, today I see a post on [Reddit](https://www.reddit.com/r/Julia/comments/157isyz/pkgjl_bug_cant_scroll_up_to_see_all_downloads/) in `r/julia` about a person have a problem Pkg.jl or even more. According to the post:

> Unfortunately, I am unable to post this as an issue to GitHub, as the smart and honorable and virtuous hero Julia moderators know that it's always better to ban people who help improve software, so they banned me everywhere except on reddit.

The reason with that hate attitude I think is about the way he want to summit an issue but his account was banned by moderators. We don't know what behind the curtain but what important in here is the issue mentioned in the post, about terminal display text when downloading a package. It's maybe helpful to someone to read the information about download dependencies in real-time.

**Comment by KristofferC** [comment]
I have no problem scrolling back up through the terminal after Pkg output. Anyway, feel free to open an issue about this if you can reproduce it.

Regarding the banning of that user, this is not a good place to discuss that.

**Reply by hungpham3112** [reply]
I believe there is reasonable though to ban that user.
